src/views/preview/preview.jsx

```diff
@@ -66,7 +68,7 @@ class Preview extends React.Component {
             extensions: [],
             favoriteCount: 0,
             loveCount: 0,
-            projectId: parts[1] === 'editor' ? 0 : parts[1],
+            projectId: parts[1] === 'editor' ? '0' : parts[1],
             addToStudioOpen: false,
             reportOpen: false
         };
```

Comment:
FYI, the call to getExtensions below should maybe be removed... I tested without it, the page loads the same and shows extensions, they just show up a moment later (because they aren't being loaded until after projectInfo is fetched)

Issue: https://github.com/LLK/scratch-www/issues/2216